TOMP-API.yaml

```diff
@@ -1283,6 +1283,24 @@ components:
         toAddress:
           $ref: "#/components/schemas/address"
 
+    bookingOptionAfterBooking:
+      type: object
+      description: A new booking, created by MaaS POST request in 'PENDING' state. The ID is generated by the TO in the 'availability-check'. The fromAddress, toAddress and the personal data should only be provided if there is an actual request for it (see conditions)
+      required:
+        - id
+        - customer
+        - fromAddress
+      properties:
+        id:
+          description: unique ID (TO's perspective) for identifying this specific available (type of) asset
+          type: string
+        customer:
+          $ref: "#/components/schemas/customer"
+        fromAddress:
+          $ref: "#/components/schemas/address"
+        toAddress:
+          $ref: "#/components/schemas/address"
+          
     bookingState:
```

Comment:
Kushagra, I don't see the purpose why you actually duplicated the bookingOption object...